<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminLibrarian</class>
 <widget class="QMainWindow" name="AdminLibrarian">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>407</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Rounded MT Bold</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Inventory_Button">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>170</width>
      <height>80</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Inventory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Members_Button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>170</y>
      <width>180</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string>Members</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>311</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Rounded MT Bold</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Librarian Main Screen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LogOut_Button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>320</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LogOut</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
